chef/cookbooks/network/recipes/default.rb

```diff
@@ -275,6 +286,10 @@ def kill_nic(nic)
       Chef::Log.info("Creating OVS bridge #{bridge} for network #{network.name}")
       Nic::OvsBridge.create(bridge)
     end
+
+    datapath_id = get_datapath_id_for_ovsbridge(bridge)
+    br.datapath_id = datapath_id unless br.datapath_id == datapath_id
```

Comment:
```network.add_ovs_bridge``` is always ```true``` during when a specific network needs an ovs_bridge (even if that bridge is already there). It's backed by a node attribute that is set e.g. by the neutron barclamp (https://github.com/crowbar/crowbar-openstack/blob/master/crowbar_framework/app/models/neutron_service.rb#L372)

I've thought about adding the datapath_id code to the ```else``` branch a few lines above to have it only executed when the bridge is created. But that would mean we would never set the id during the upgrade from Cloud 6. So I opted for the current approach